File: twit.js
const twit = require('twitter');
const bodyParser = require('body-parser');
const app = twitter();

const tweetFrom = 'TwithubB';

var client = new Twitter({
	consumer_key: 'G61bMjUHHxLrHzTnrCSfxI4IeOLtOSFXJUBM51xKDmJ5rlDAL8', 
	consumer_secret: 'G61bMjUHHxLrHzTnrCSfxI4IeOLtOSFXJUBM51xKDmJ5rlDAL8',
	access_token_key: '704876576337477636-y1ABqx4EJN2laossezoedWlSKjs6NHx', 
	access_token_secret: 'nils8jhtwyD5jQtP4DOLAVRLP9QrnZjfxB0VWdfGLx3Dy'
});

var params = {screen_name: 'TwithubB'};
client.get('statuses/user_timeline', params, function(error, tweets, response){
	if (!error) {
		console.log(tweets);
	}
};

client.post('statuses/update', {status: "Ben's a dick"}, function(error, tweet, response){
	if(error) throw error;
	console.log(tweet)
	console.log(response);
});

File: twit.rb
require 'rubygems'
require 'oauth'
require 'json'

consumer_key = OAuth::Consumer.new(
  "j5oWVtmbD3MnrGtRk0jMOEI4z",
    "VfoxZbtrTjeSPVOkuzsmjUB8c6TmIHKsZIZr83MSo0CpN1x72F")
    access_token = OAuth::Token.new(
      "704876576337477636-9wsUP6ZuWHsbpHhnSyiOQIkoxd5OalJ",
        "IZ06WkLDpmCzvLXf6tTKVYl893xPIV39PK7wUCHovf8qc")

	baseurl = "https://api.twitter.com"
	path    = "/1.1/statuses/update.json"
	address = URI("#{baseurl}#{path}")
	request = Net::HTTP::Post.new address.request_uri
	request.set_form_data(
	  "status" => "This tweet was sent using twit.rb",
	  )

	  http             = Net::HTTP.new address.host, address.port
	  http.use_ssl     = true
	  http.verify_mode = OpenSSL::SSL::VERIFY_PEER

	  request.oauth! http, consumer_key, access_token
	  http.start
	  response = http.request request

	  tweet = nil
	  if response.code == '200' then
	    tweet = JSON.parse(response.body)
	      puts "Successfully sent #{tweet["text"]}"
	      else
	        puts "Could not send the Tweet! " +
		  "Code:#{response.code} Body:#{response.body}"
end
